fixtures/fixtures.js

```diff
@@ -90,12 +102,27 @@ Fixtures.apply = function (mock) {
     mock.onGet(/\/traders\/2\/vouchers/).reply(200, this.traderVouchers["2"]);
 
     mock.onPost('/vouchers').reply(function (request) {
```

Comment:
It's GET the list of vouchers that are already approved; that should come back as an object list of correctly assigned vouchers.. 
The format above is part of the POST response, which tells you the result of the action you've done (i.e. "please tell me if this list of 1(or more) vouchers were progressed correctly")